### PR TITLE
fr-translation

### PR DESCRIPTION
**Pull request by BibichePython**

### CODE REVIEW

File: translations/fr/info/task_description.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+</p>
+<p>
+Votre vache est jeune et ne peut utiliser que le vocabulaire que vous lui avez appris.
+Non seulement votre vache parle, mais elle présente ses paroles au fameux format Tux 
+
+    (<a href="http://en.wikipedia.org/wiki/Cowsay">wiki/Cowsay</a>) si on le lui demande gentiment.
+</p>
+
+<div>
+Un texte vous est donné, et votre fonction doit mettre le discours de votre vache au format "cowsay" ("parole de vaches" ).
```
Comment (Cjkjvfnby) [inline translations/fr/info/task_description.html]: `("parole de vaches" )`  unrequired space

File: translations/fr/info/task_description.html [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+Décrivons les règles de cette mise en forme :
+
+    <ul class="rules">
+        <li>
+        La vache est toujours la même, seule sa citation change.
```
Comment (Cjkjvfnby) [inline translations/fr/info/task_description.html]: I prefer to add extra indent to tag inner content
```html
<li>
    La vache est toujours la même, seule sa citation change.
</li>
```

File: translations/fr/info/task_description.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        </li>
+        <li>
+        Le programme pour la console "cowsay" a un comportement étrange dans certains cas spéciaux, qui ne seront pas testés ici.</li>
+    </ul>
+</div>
+
+<div>
+    <pre>
+     _________________________________
+    / Dog goes woof                   \
```
Comment (Cjkjvfnby) [inline translations/fr/info/task_description.html]: If you have suitable french poem feel free to add it here.

File: translations/fr/info/task_description.html [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                (__)\       )\/\
+                    ||----w |
+                    ||     ||
+    </pre>
+    <strong>What does the cow say? (Que dit la vache ?)</strong>
+</div>
+
+<br>
+<p>
+    <strong>Input: </strong> Un texte (string).
```
Comment (Cjkjvfnby) [inline translations/fr/info/task_description.html]: `<strong>Input: </strong>` -> `<strong>Input:</strong>`  no need to have two spaces here.

same for `Output`

**Comment by Cjkjvfnby** [comment]
Hi, thank you for translation.

I totally forget what should I do after merge of PR to deploy code to checkio :(

**Comment by BibichePython** [comment]
Hi !
Thank you so much for your comments ! 
I think I have fixed everything.
I love this task !
Thank you for all.

I suppose you could contact Alex for help about the next step after merge. I have no idea about what to do...

**Pull request merged**